components/x-gift-article/storybook/error-response.js

```diff
@@ -19,6 +17,7 @@ exports.m = module
 
 exports.fetchMock = (fetchMock) => {
 	fetchMock
+		.restore()
```

Comment:
is it worth doing this in a global setup file?

Reply:
calling the `restore` method? I am calling it on each story so that the mocks are isolated to each